go.mod

```diff
@@ -17,13 +17,13 @@ require (
 	github.com/stretchr/testify v1.8.2
 	google.golang.org/grpc v1.54.0
 	google.golang.org/protobuf v1.30.0
-	k8s.io/api v0.26.3
-	k8s.io/apimachinery v0.26.3
-	k8s.io/apiserver v0.26.1
-	k8s.io/client-go v0.26.1
-	k8s.io/component-base v0.26.1
-	k8s.io/component-helpers v0.26.1
-	k8s.io/csi-translation-lib v0.26.3
+	k8s.io/api v0.27.0-rc.0
+	k8s.io/apimachinery v0.27.0-rc.0
```

Comment:
searching for where the `ServerSideFieldValidation` feature gate is defined:
```
./vendor/k8s.io/apiserver/pkg/features/kube_features.go:        ServerSideFieldValidation: {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.29
./vendor/k8s.io/kubernetes/pkg/features/kube_features.go:       genericfeatures.ServerSideFieldValidation: {Default: true, PreRelease: featuregate.Beta},
```

I see that the same feature gate is defined but with different PreRelease values.

I think the kubernetes dependency at L36 also needs to be updated.

Reply:
but in 1.27 branch it is the same value: https://github.com/kubernetes/kubernetes/blob/release-1.27/pkg/features/kube_features.go#L1194

Reply:
I mean this line should also be updated to 1.27: https://github.com/kubernetes-csi/external-provisioner/blob/master/go.mod#L36

The script is supposed to do this; seems like there's a bug